File: android2_lesson03/src/java/java/com/example/ama/android2_lesson03/utils/ResourceUtils.java
package com.example.ama.android2_lesson03.utils;

import android.location.Location;

import com.example.ama.android2_lesson03.PocketMap;
import com.example.ama.android2_lesson03.R;

public class ResourceUtils {

    public static String formatLocationCoordinates(Location location) {
        return String.format(PocketMap.getInstance().getResources().getString(R.string.widget_latlng_format),
                location.getLatitude(), location.getLongitude());
    }

    public static String getStringResource(int resId) {
        return PocketMap.getInstance().getResources().getString(resId);
    }
}

File: android2_lesson03/src/kotlin/java/com/example/ama/android2_lesson03/utils/ResourceUtils.kt
package com.example.ama.android2_lesson03.utils

import android.location.Location
import com.example.ama.android2_lesson03.PocketMap
import com.example.ama.android2_lesson03.R

object ResourceUtils {

    fun formatLocationCoordinates(location: Location) =
            String.format(PocketMap.instance.resources.getString(R.string.widget_latlng_format),
                    location.latitude, location.longitude)
}